[2025-03-11T09:09:51.811+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-11T09:09:52.676+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: validate_all_tables.customers manual__2025-03-11T09:09:41.505828+00:00 [queued]>
[2025-03-11T09:09:52.721+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: validate_all_tables.customers manual__2025-03-11T09:09:41.505828+00:00 [queued]>
[2025-03-11T09:09:52.833+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-11T09:09:52.980+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): customers> on 2025-03-11 09:09:41.505828+00:00
[2025-03-11T09:09:53.064+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=8183) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-11T09:09:53.104+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'validate_all_tables', 'customers', 'manual__2025-03-11T09:09:41.505828+00:00', '--job-id', '224', '--raw', '--subdir', 'DAGS_FOLDER/validate_all_tables_dag.py', '--cfg-path', '/tmp/tmp98pa5j0w']
[2025-03-11T09:09:53.224+0000] {standard_task_runner.py:105} INFO - Job 224: Subtask customers
[2025-03-11T09:09:53.160+0000] {standard_task_runner.py:72} INFO - Started process 8195 to run task
[2025-03-11T09:09:53.717+0000] {task_command.py:467} INFO - Running <TaskInstance: validate_all_tables.customers manual__2025-03-11T09:09:41.505828+00:00 [running]> on host efacfdd0be48
[2025-03-11T09:09:54.364+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='validate_all_tables' AIRFLOW_CTX_TASK_ID='customers' AIRFLOW_CTX_EXECUTION_DATE='2025-03-11T09:09:41.505828+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-11T09:09:41.505828+00:00'
[2025-03-11T09:09:54.367+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-11T09:09:54.369+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-11T09:09:54.370+0000] {logging_mixin.py:190} INFO - Current task name:customers state:running start_date:2025-03-11 09:09:52.683527+00:00
[2025-03-11T09:09:54.372+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables and current dag run status:running
[2025-03-11T09:09:54.373+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-11T09:09:54.435+0000] {logging_mixin.py:190} INFO - Validation successful. Found 1445 customer records.
[2025-03-11T09:09:54.436+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-11T09:09:54.471+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-11T09:09:54.483+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=validate_all_tables, task_id=customers, run_id=manual__2025-03-11T09:09:41.505828+00:00, execution_date=20250311T090941, start_date=20250311T090952, end_date=20250311T090954
[2025-03-11T09:09:54.637+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-11T09:09:54.652+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-11T09:09:54.674+0000] {logging_mixin.py:190} INFO - Dag name:validate_all_tables queued_at:2025-03-11 09:09:41.563044+00:00
[2025-03-11T09:09:54.678+0000] {logging_mixin.py:190} INFO - Task hostname:efacfdd0be48 operator:PythonOperator
[2025-03-11T09:09:54.747+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-11T09:09:54.776+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
